spec/lib/describe_indexer_spec.rb

```diff
@@ -121,27 +122,21 @@
 
     context "files" do
       it "stores file detailed information" do
-        files = JSON.parse(indexed_record['files_ss'])
-        file1 = files.find { |file| file["name"] == "file1.jpg" }
-        file2 = files.find { |file| file["name"] == "file2.txt" }
-        file3 = files.find { |file| file["name"] == "file3.txt" }
-        expect(file1["size"]).to eq "316781"
+        files = JSON.parse(indexed_record['pdc_describe_json_ss'])['files']
+        file1 = files.find { |file| file["filename"] == "10.80021/3m1k-6036/122/file1.jpg" }
+        file2 = files.find { |file| file["filename"] == "10.80021/3m1k-6036/122/file2.txt" }
+        file3 = files.find { |file| file["filename"] == "10.80021/3m1k-6036/122/folder-a/file3.txt" }
+        expect(file1["size"]).to eq 316_781
         expect(file1["url"]).to eq "https://g-5beea4.90d4e.bd7c.data.globus.org/pdc-describe-staging-postcuration/10.80021/3m1k-6036/122/file1.jpg"
-        expect(file2["size"]).to eq "396003"
-        expect(file3["name"]).to eq "file3.txt"
-        expect(file3["full_name"]).to eq "10.80021/3m1k-6036/122/folder-a/file3.txt"
-      end
-      it "excludes PDC preservation files" do
```

Comment:
We removed this test because since August 2023 we are not longer importing preservation files (those under `/princeton_data_commons/`) into Discovery. See https://github.com/pulibrary/pdc_describe/pull/1387